sdk/containerapps/Azure.ResourceManager.AppContainers/src/Generated/Models/JobScale.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.AppContainers.Models
+{
+    /// <summary> Scaling configurations for event driven jobs. </summary>
+    public partial class JobScale
+    {
+        /// <summary> Initializes a new instance of JobScale. </summary>
+        public JobScale()
+        {
+            Rules = new ChangeTrackingList<JobScaleRule>();
+        }
+
+        /// <summary> Initializes a new instance of JobScale. </summary>
+        /// <param name="pollingInterval"> Interval to check each event source in seconds. Defaults to 30s. </param>
+        /// <param name="minExecutions"> Minimum number of job executions that are created for a trigger, default 0. </param>
+        /// <param name="maxExecutions"> Maximum number of job executions that are created for a trigger, default 100. </param>
+        /// <param name="rules"> Scaling rules. </param>
+        internal JobScale(int? pollingInterval, int? minExecutions, int? maxExecutions, IList<JobScaleRule> rules)
+        {
+            PollingInterval = pollingInterval;
+            MinExecutions = minExecutions;
+            MaxExecutions = maxExecutions;
+            Rules = rules;
+        }
+
+        /// <summary> Interval to check each event source in seconds. Defaults to 30s. </summary>
+        public int? PollingInterval { get; set; }
```

Comment:
What's the unit? `InSeconds` or `InMinutes` or else? Better to append the unit to the name.

Reply:
Agreed. Can we make this change without breaking swagger spec? We don't want to manually make the change for every.net sdk generation.